less-discourse/README.md

```diff
@@ -0,0 +1,31 @@
+
+### Menu on the Discourse Forum
+
+The goal was to rebuild the exact same menu in both UI and on the Discourse Forum.
+Only a few icons change (to differenciate between the already-too-similar-ones on Discourse) and the HTML for the menu on Discourse had to be adapted.
+
+If you make any changes to the `sidemenu.html` or `sidemenu.less`, make sure you update the Discourse's menu too.
```

Comment:
``sidemenu.html`` does not exist. You have named it ``menu.html``. Perhaps you should rename it.
Using the full paths would help to figure out what files you are talking about.